lib/server.js

```diff
@@ -28,7 +31,7 @@ module.exports = Server;
 
 Server.prototype.bind = function(host) {
 	var _this = this;
-    this.server.bind(67, host, function() { 
```

Comment:
Seems like a space was added at the end here -- can you remove that from the patch?